tests/utils/helper_funcs.py

```diff
@@ -8,68 +11,39 @@ def delete_records(session, records):
+def delete_all_records(session):
+    for table in [DashboardSource, ResearchSource, Country, City, State, TestManufacturer, AntibodyTarget, CityBridge,
+                  StateBridge, TestManufacturerBridge, AntibodyTargetBridge]:
+        session.query(table).delete()
+    session.commit()
+    return
 
-    # Insert joined record into population group table
-    factories.population_group_factory(session, population_group_id=population_group_id)
 
-    # Insert joined record into state bridge table
-    new_state_record = factories.state_bridge_factory(session, source_id=source_id)
+# Inserts a complete set of mocked dashboard records
+# each sharing the same country, city, state, approving regulator, test manufacturer, and study
+def insert_full_recordset(num_records=1, study_name="study"):
+    # Insert records into non dashboard_source/research_source/bridge tables
+    new_antibody_target_record = factories.AntibodyTargetFactory()
+    antibody_target_id = new_antibody_target_record.antibody_target_id
+    new_city_record = factories.CityFactory()
+    city_id = new_city_record.city_id
+    new_country_record = factories.CountryFactory()
+    country_id = new_country_record.country_id
+    new_state_record = factories.StateFactory()
     state_id = new_state_record.state_id
-
-    # Insert joined record into state table
-    factories.state_factory(session, state_id=state_id)
-
-    # Insert joined record into specimen type bridge table
-    new_specimen_type_record = factories.specimen_type_bridge_factory(session, source_id=source_id)
-    specimen_type_id = new_specimen_type_record.specimen_type_id
-
-    # Insert joined record into specimen type table
-    factories.specimen_type_factory(session, specimen_type_id=specimen_type_id)
-
-    # Insert joined record into test type bridge table
-    new_test_type_record = factories.test_type_bridge_factory(session, source_id=source_id)
-    test_type_id = new_test_type_record.test_type_id
-
-    # Insert joined record into test type table
-    factories.test_type_factory(session, test_type_id=test_type_id)
-
-    # Insert joined record into test manufacturer bridge table
-    new_test_manufacturer_record = factories.test_manufacturer_bridge_factory(session, source_id=source_id)
+    new_test_manufacturer_record = factories.TestManufacturerFactory()
     test_manufacturer_id = new_test_manufacturer_record.test_manufacturer_id
 
-    # Insert joined record into test manufacturer table
-    factories.test_manufacturer_factory(session, test_manufacturer_id=test_manufacturer_id)
+    # Insert records into dashboard_source, research_source, and bridge tables
+    for i in range(num_records):
+        source_id = uuid.uuid4()
+        # Ensure all are dashboard_primary estimates so that estimate prioritization
+        # always returns 1 estimate per study
+        factories.DashboardSourceFactory(source_id=source_id, country_id=country_id, study_name=study_name)
```

Comment:
again, realism here -- what would a realistic set of estimates look like? prob 1x dash primary, 1x academic primary, then 10x random other estimates with varying other things... not sure how important this is for the sort of testing we're doing, though!